ITC205_BaseLine_Code/src/Loan.java

```diff
@@ -72,7 +72,7 @@ public book Book() {
 
 
 	public void Loan() {

```

Comment:
D3- Method names can be made more meaningful on following lines.

Line 64: line 64: public member Member -> public member getMember
line 69: public book Book -> public book getBook
line 74: public void Loan -> public void dischargeLoan

(Note: was unable to comment lines 64 & 69)